.env.example

```diff
@@ -1,17 +1,8 @@
 # Database - change to fit your database
-DB_HOST = 127.0.0.1 
```

Comment:
This file should be unchanged, just like it is on master now: https://github.com/Saidemm/Meal-Sharing/blob/master/.env.example

Reply:
Thanks for checking it.